### DISCUSSION TITLE
如何将系统粘贴，粘贴到vim中

### DISCUSSION THREAD

**Discussion by wxzvim** [ayamir/nvimdots#825] [Q&A]
配置好neovim，打开总是鼠标模式，无法复制系统粘贴，咋办？
临时解决办法，是 :set mouse=  但每次都得这样，，好麻烦，，有什么办法可以解决？

**Comment by wxzvim** [comment]
系统是centos 7的，，终端是xshell，是不是没有适配这种情况？

**Comment by Jint-lzxy** [comment]
Can you help fill in the [issue form](https://github.com/ayamir/nvimdots/issues/new?assignees=&labels=bug&template=bug_report.yml)? This discussion provides too little information, and we can't _infer_ the problem from a simple description. Instead, we need **steps** that could **reproduce** this issue (describe _what you have done_ in an ordered list) so that we can help you with this.

**Reply by wxzvim** [reply]
how to turn off mouse mode ?

**Reply by wxzvim** [reply]
i want to shutdown mouse mode ,it is not complex ,I  want to know   how to configure ?

**Comment by wxzvim** [comment]
找到方法了，，option配置，，加上  mouse=false 就行了

**Comment by DevasOps** [comment]
应该是设置成 mouse=‘’  这样才行
